src/main/java/pl/javastart/task/Circle.java

```diff
@@ -0,0 +1,21 @@
+package pl.javastart.task;
+
+public class Circle {
+    private double radius;
+
+    public Circle(double radius) {
+        this.radius = radius;
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public void setRadius(double radius) {
+        this.radius = radius;
+    }
+
+    public double calculateCircleArea() {
+        return 3.14159 * (getRadius() * getRadius());
```

Comment:
Jeśli masz bezpośredni dostęp do pola to korzystaj z tego, więc tutaj lepiej: 
```java
 return 3.14159 * radius * radius
```
a jeszcze lepiej to skorzystać ze stałej `Math.PI`

Reply:
Tak oczywiste, że aż wstyd za gapiostwo... ;)